lib/biokbase/catalog/Client.py

```diff
@@ -12,7 +12,7 @@
 try:
     # baseclient and this client are in a package
     from .baseclient import BaseClient as _BaseClient  # @UnusedImport
-except:
+except ImportError:
```

Comment:
Is it safe to assume the changes in Client.py are just new autogenerated code from the SDK?

Reply:
Yeah, also the changes to authclient and baseclient